src/utils/exec.ts

```diff
@@ -8,6 +9,23 @@ import npmWhich from 'npm-which';
 import { isErrorWithCode } from './error';
 import { log } from './logging';
 
+class YarnWarningFilter extends stream.Transform {
+  _transform(
+    chunk: any,
+    _encoding: BufferEncoding,
+    callback: stream.TransformCallback,
+  ) {
+    const str = Buffer.from(chunk).toString();
+
+    // Filter out annoying deprecation warnings that users can do little about
+    if (!str.startsWith('warning skuba >')) {
```

Comment:
When I read the PR description I was excited to figure out which Yarn argument did this.